app/components/rive-animation.ts

```diff
@@ -0,0 +1,157 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { Fit, Layout, Rive } from '@rive-app/canvas';
+import * as Sentry from '@sentry/ember';
+
+interface RiveAnimationSignature {
+  Element: HTMLDivElement;
+  Args: {
+    src: string;
+  };
+}
+
+export default class RiveAnimationComponent extends Component<RiveAnimationSignature> {
+  animationInterval: number | null = null;
+  container: HTMLElement | null = null;
+  resizeObserver: ResizeObserver | null = null;
+  @tracked riveInstance: Rive | null = null;
+  @tracked containerWidth: number = 0;
+  @tracked containerHeight: number = 0;
```

Comment:
See if we can avoid tracking `containerHeight` / `containerWidth` - and just rely on natural HTML sizing